explainer.md

```diff
@@ -578,10 +623,12 @@ interface VRDevice : EventTarget {
 
 dictionary VRSessionCreateParametersInit {
   boolean exclusive = true;
+  HTMLCanvasElement outputCanvas = null;
```

Comment:
Out of curiosity, why HTMLCanvasElement rather than the ImageBitmapRenderingContext given the hard requirement listed above?